src/tracks/InstrumentTrack.cpp

```diff
@@ -1130,6 +1156,37 @@ void InstrumentTrackView::midiConfigChanged()
 
 
 
+QMenu * InstrumentTrackView::createFxMenu(QString title, QString newFxLabel)
```

Comment:
I would imagine an FX menu would live as part of GUI code within some mixer class.  It may even serve as a static utility-type function that constructs the menu as well as its items based on the list/object passed in, set up the parent properly and return a reference to the new menu although I could be off.

Having this code in the InstrumentTrackView does seem slightly out of place in hindsight.